samples/bluetooth/iso_connected_benchmark/README.rst

```diff
@@ -0,0 +1,111 @@
+.. _iso_broadcast_benchmark:
+
+Bluetooth: Throughput
+#####################
+
+The ISO Broadcast Benchmark sample tests the quality of connected ISO channels
+in terms for packet and sync loss.
+
+
+Overview
+********
+
+The sample transmits data between the *central* and the *peripheral*
+and measures the quality of the sync.
+
+The application is used as both a central and a peripheral, and the mode
```

Comment:
mode?  Whether the application is in the central role or the peripheral role?

Not used as both at the same time, I guess?

The application can be used as either a central or a peripheral, and can easily switch between these roles.

Reply:
Correct.

Reply:
Just wondering whether it would be better to actually use the word "role", rather than "mode", when talking about switching to the other role.